metals/src/main/scala/scala/meta/internal/tvp/TreeViewNode.scala

```diff
@@ -24,18 +24,21 @@ case class TreeViewNode(
 }
 
 object TreeViewNode {
-  def fromCommand(command: Command): TreeViewNode =
+  def fromCommand(
+      command: Command,
+      icon: String = TreeViewNode.command
+  ): TreeViewNode =
     TreeViewNode(
       viewId = "commands",
       nodeUri = s"metals://command/${command.id}",
       label = command.title,
       command = MetalsCommand(
         command.title,
-        command.id,
+        "metals." + command.id,
```

Comment:
Somehow this didn't work with the new command, since the ones declared all have the `metals.` prefix

This might not be needed, but I haven't been able to make the new command work otherwise.

I think we could also just use an echo command for all server commands here.